lib/much-rails.rb

```diff
@@ -16,6 +16,7 @@
 require "much-rails/has_slug"
 require "much-rails/input_value"
 require "much-rails/json"
+require "much-rails/layout"
```

Comment:
I'd like to have a `require "much-rails/view_models" and corresponding file. That way the gem requires in the stuff it offers automatically.